.ci.yaml

```diff
@@ -7,6 +7,9 @@
 #  * https://github.com/flutter/cocoon/blob/master/CI_YAML.md
 enabled_branches:
   - master
+  - dev
```

Comment:
fyi @christopherfujino this is my plan to make sure we can generate builders for these branches, but you'll still need to pass the specific release branch in conductor to ensure tests are run

Reply:
I'm a little fuzzy on how that will work, but I'm guessing it'll be spelled out in your one-pager?